chad/cards-template1b.js

```diff
@@ -0,0 +1,125 @@
+// Error-detecting version
```

Comment:
Good job. Don't forget to write your own assertion tests so you have a strong grasp on how they work!